sdk/storage/azure_storage_blob/tests/blob_client.rs

```diff
@@ -15,19 +20,41 @@ async fn test_get_blob_properties() -> Result<(), Box<dyn Error>> {
     let credential = DefaultAzureCredentialBuilder::default().build()?;
 
     // Act
+    let container_client = BlobContainerClient::new(
+        &endpoint,
+        String::from("testcontainer1"),
+        credential.clone(),
+        None,
+    )?;
+    container_client.create_container(None).await?;
+
     let blob_client = BlobClient::new(
         &endpoint,
-        String::from("testcontainer"),
+        String::from("testcontainer1"),
         String::from("test_blob.txt"),
         credential,
         Some(BlobClientOptions::default()),
     )?;
+    let data = b"hello rusty world";
+    blob_client
+        .upload_blob(
+            RequestContent::from(data.to_vec()),
+            true,
+            data.len() as i64,
```

Comment:
This is possibly unsafe code as we are going from `usize` to `i64` which could cause overflow. We should consider adding safe conversion logic to the handwritten API, but we need to figure out the most cheapest and safest way to calculate len.

Reply:
For now I will leave the handwritten API matching the signature of the underlying generated code-- refactored tests to use `try_from` which will bubble up any conversion errors instead of silently swallowing like `as`.